['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmputsewxha/solution_12.py", line 102, in <module>\n    print(Solution().minimumDiameterAfterMerge(edges1 = [[0,1],[1,2],[2,3],[3,4],[4,5],[5,6],[6,7],[7,8],[8,9],[9,10],[10,11],[11,12],[12,13],[13,14],[14,15],[15,16],[16,17],[17,18],[18,19],[19,20]], edges2 = [[21,22],[22,23],[23,24],[24,25],[25,26],[26,27],[27,28],[28,29],[29,30],[30,31],[31,32],[32,33],[33,34],[34,35],[35,36],[36,37],[37,38],[38,39],[39,40]]))\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmputsewxha/solution_12.py", line 81, in minimumDiameterAfterMerge\n    graph_two[u].append(v)\nIndexError: list index out of range\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmputsewxha/solution_14.py", line 102, in <module>\n    print(Solution().minimumDiameterAfterMerge(edges1 = [[0,1],[0,2],[1,3],[1,4],[2,5],[2,6],[3,7],[3,8],[4,9],[4,10],[5,11],[5,12],[6,13],[6,14],[7,15],[7,16],[8,17],[8,18],[9,19],[9,20],[10,21],[10,22]], edges2 = [[23,24],[24,25],[25,26],[26,27],[27,28],[28,29],[29,30],[30,31],[31,32],[32,33],[33,34],[34,35],[35,36],[36,37],[37,38],[38,39]]))\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmputsewxha/solution_14.py", line 81, in minimumDiameterAfterMerge\n    graph_two[u].append(v)\nIndexError: list index out of range\n']